### DISCUSSION TITLE
Report by cashier

### DISCUSSION THREAD

**Discussion by Zakariae-BOUCHFIRA** [Blair2004/NexoPOS#1741] [Ideas]
Hello,
I want to add a report by the cashier, I need them at the end of every day the cashiers come and print their report containing the order that they pass with the total of sales and the total of cash sales and paid by visa card in the last 24hour I found the method already exists but there is no view.
Thank you

**Comment by Blair2004** [comment]
Hi, thank you for joining us.
Do you want to create a report yourself?

**Comment by Blair2004** [comment]
Are we talking about this filter ?
![screenshot-nimbusweb me-2024 01 02-23_19_13](https://github.com/Blair2004/NexoPOS/assets/5265663/9e52bf06-f26b-4c76-80d0-cfed75616b17)

**Comment by Zakariae-BOUCHFIRA** [comment]
I need to add to this report the total price grouped by payment type (cash, visa card)
